src/database/sql/sequelize/models/biometrics/blood.oxygen.saturation.model.ts

```diff
@@ -10,20 +10,18 @@ import {
     PrimaryKey,
     Length,
     ForeignKey,
-    BelongsTo,
-    IsDecimal,
-    IsDate } from 'sequelize-typescript';
+    IsDate,
+} from 'sequelize-typescript';
 
 import { v4 } from 'uuid';
 import User from '../user.model';
-import Person from '../person.model';
 
 ///////////////////////////////////////////////////////////////////////
 
 @Table({
     timestamps      : true,
     modelName       : 'BloodOxygenSaturation',
-    tableName       : 'biometrics_blood_oxygen_saturation',
+    tableName       : 'blood_oxygen_saturation',
```

Comment:
**biometrics_blood_oxygen_saturation** was correct name. Only for modelName we do not want Biometrics in there as it is already fully qualified. In database, we need additional qualification with the help of prefix (i.e. **biometrics_**).